mozilla/fx-private-relay
# Rename to gunicorn.conf.py

`gunicorn.conf.py` has been the default since 17.5 (2013-07-03), and gunicorn starts with a warning if the config file doesn't end in `.py`.

# How to test:

On `main`, you can run this to see the error:

```sh
gunicorn --config gunicorn.conf privaterelay.wsgi:application
```

You can also see it in the GCP logs with [this query](https://cloudlogging.app.goo.gl/VsamKDayAaizdur96):

```
textPayload=~"^!!!"
severity=ERROR
labels."k8s-pod/app_kubernetes_io/name"="fxprivaterelay"
```

On this branch, you can run without the `--config` argument and get no error:

```sh
gunicorn privaterelay.wsgi:application
```
